lib/loaders/typescript.js

```diff
@@ -11,6 +11,7 @@
 
 const loaderFeatures = require('../loader-features');
 const babelLoader = require('./babel');
+const forkedTypesPluginUtil = require('../plugins/forked-ts-types');
```

Comment:
When you require this file, it requires `fork-ts-checker-webpack-plugin`. And THAT causes the big Node error in case `fork-ts-checker-webpack-plugin` isn't installed. What you can do is move this `require` statement down - it doesn't need to be on top:

if (webpackConfig.useForkedTypeScriptTypeChecking) {
    loaderFeatures.ensureLoaderPackagesExist('forkedtypecheck');
    
    const forkedTypesPluginUtil = require('../plugins/forked-ts-types');
}

require is most commonly done on top, but it's not a requirement.